<launch>
	<node pkg="tracker" type="tracker" name="tracker" output="screen">
	<param name="ip" value="172.17.0.1" />
	<param name="port" value="5556" />
        <param name="run_optical_flow" value="0" />
        <param name="track_run_iter" value="10" />
        <param name="init_timeout" value="20" />
	</node>
</launch>
